js/main.js

```diff
@@ -518,25 +511,58 @@ function generateHTMLCalendarDay(square, dateObj, isCurrentMonth, dayObj, curren
   }
 
   // Travel
+  var travelAdded = false;
   if (dayObj) {
     if (dayObj.travel) {
-      if (dayObj.travel !== currentTravel.location) {
-        currentTravel.style = 'travel-' + (parseInt(currentTravel.style[7]) + 1);
-        if (currentTravel.style[7] === '5') {
-          currentTravel.style = 'travel-1';
+      if (dayObj.travel !== data.homeTown) {
+        travelAdded = true;
+        if (dayObj.travel !== currentTravel.location) {
+          currentTravel.location = dayObj.travel;
+          currentTravel.style = 'travel-' + (parseInt(currentTravel.style[7]) + 1);
```

Comment:
I think `currentStyle.style[7]` would be good to store in a variable that describes its purpose, this is a bit hard to read 🤔